src/main.cpp

```diff
@@ -30,6 +30,9 @@ static void initMisc() {
 
     // Enable z-buffer test
     glEnable(GL_DEPTH_TEST);
+
+    // Seed the PRNG with the current time for any random elements in the world
+    std::srand(std::time(NULL));

```

Comment:
I moved it into main from GameWorld is all, not really used anymore anyway, but if we load another level we don't need to reseed anything so I moved it.